rotkehlchen/accounting/history_base_entries.py

```diff
@@ -21,26 +23,6 @@
 T = TypeVar('T', bound=HistoryBaseEntry)
 
 
-def history_base_entries_iterator(
```

Comment:
This I see and am wondering what the fuck were we thinking. This seems to be just for typing and error logging. And to re-ierate the iterator every time it is called?

Reply:
is not re-itearting it, just yielding. So consume the next event, check and yield the consumed event